src/main/java/org/scijava/log/StderrLogService.java

```diff
@@ -51,12 +51,25 @@
 @Plugin(type = Service.class, priority = Priority.LOW_PRIORITY)
 public class StderrLogService extends AbstractLogService {
 
-	private final LogFormatter formatter = new DefaultLogFormatter();
+	public StderrLogService() {
+		addAllChannelsLogListener(new PrintListener());
+	}
+
+	private static class PrintListener implements LogListener {
+
+		private final LogFormatter formatter = new DefaultLogFormatter();
+
+		private final Logger dummySource = new DefaultLogger(); // FIXME add "Logger source" as parameter to messageLogged
```

Comment:
Sorry, I don't understand this commit. We do _not_ want to emit _all_ channels to stderr/stdout... only the default channel. This was explicitly part of the design decided upon by @fjug and myself. The idea is that you may want to create some additional channels which do not appear on stdout, but which only your particular code is interested in consuming.